roles/cloudprober/tasks/main.yml

```diff
@@ -1,4 +1,35 @@
 # Setup Cloudprober
+
+# Create OnePassword Item to store required info
+- name: Loki Data
+  set_fact:
+    loki_password: "{{ lookup('dbildungscloud.onepwd.onepwd', secret_name='loki',vault=vault, field='password', credentials=op_credentials) }}"
+    loki_user: "{{ lookup('dbildungscloud.onepwd.onepwd', secret_name='loki',vault=vault, field='username', credentials=op_credentials) }}"
+  tags: cloudprober
```

Comment:
i think i would prefer it when these taaks are optinal via a when condition. Furthermore it may be useful to add the the defaults/main.yml not only the secret name, but also leave a comment which passwords may be added